azure-pipelines.yml

```diff
@@ -136,6 +153,23 @@ jobs:
       publishVstsFeed: '2cac8390-d7aa-4320-b481-1a32c53de85b'
       allowPackageConflicts: true
 
+  - task: PowerShell@1
+    condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+    displayName: 'Push VSIX'
+    inputs:
+      scriptType: inlineScript
+      workingFolder: 'bin\$(BuildConfiguration)'
+      inlineScript: |
+        # Setting these environment variables because they are used in the script for the upload endpoint: 
+     
+        $env:APPVEYOR_REPO_PROVIDER = "github"
+        $env:APPVEYOR_REPO_NAME = "microsoft/dotnet-apiport"
+     
+        Invoke-WebRequest "https://raw.github.com/madskristensen/ExtensionScripts/master/AppVeyor/vsix.ps1" -UseBasicParsing | Invoke-Expression
```

Comment:
What does this script do? Mads is clearly a reasonable person, but this seems like a liability in an official build. Should we host our own copy of this script?

Reply:
Good point. However, I hope there is more official channel hosting the same script instead maintaining our own copy. Following with with Mads to get more advice.

Reply:
There is no other more official channel to get the script other than https://raw.github.com/madskristensen/. Create local script with subset of only what we need.